apps/s_client.c

```diff
@@ -2013,10 +2022,18 @@ int s_client_main(int argc, char **argv)
                    "-dane_tlsa_domain option.\n", prog);
         goto end;
     }
+#ifndef OPENSSL_NO_DTLS
+    if (isdtls && tfo) {
+        BIO_printf(bio_err, "%s: DTLS does not support the -tfo option\n", prog);
+        goto end;
+    }
+#endif
```

Comment:
Don't we need smth like this in s_server?

Reply:
Added check in fixup.

Reply:
How the heck is this failing CI now on `#include "e_os.h"` when all I did was add a fixup commit for `s_server`?
I'm not seeing any more recent changes on master that would impact this?

Reply:
Looks like d5f9166bac on master broke it.